packages/api/types.d.ts

```diff
@@ -258,6 +261,46 @@ declare global {
 		 */
 		quote: string
 	}
+
+	type KrakenAssetList = {
+		error: string[]
+		result: {
+			[key: string]: {
+				altname: string
+				aclass: string
+				decimals: number
+				display_decimals: number
+				status: string
+			}
+		}
+	}
+
+	type BlueCache = {
```

Comment:
i think this interface should stay in cache, it's the blueprint of the cache module and no other modules is going to use it